Menu/Builder.lua

```diff
@@ -1625,6 +1646,8 @@ function Builder:CreateClassBarOptions(parent, widgetName)
         const.OPTION_KIND.SAME_SIZE_AS_HEALTH_BAR)
     self:AnchorRightOfCB(f.sameSizeAsHealthBar, f.verticalFill)
 
+    f.hideOutOfCombat = self:CreateHideOutOfCombat(f, widgetName)
+
```

Comment:
```suggestion
```
This should be removed, since it's not actually pointed to anything it's essentially just creating an invisible checkbox - and it's already added in the `ClassBar` widget file.